packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -1,9 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 export enum TelemetryEvent {
+  QuickStart = "quick-start",
+
+  Samples = "samples",
+
+  Documentation = "documentation",
+
   LoginStart = "login-start",
   Login = "login",
 
+  LoginAzure = "login-azure",
```

Comment:
Let's use "login-start" and "login" for event name, and add "account-type" property to indicate account type.